<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HotKey Changer</string>
  </property>
  <widget class="QPushButton" name="hotkeyButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Arial;
</string>
   </property>
   <property name="text">
    <string>F6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;
font-family: Arial;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press button to change&lt;/p&gt;&lt;p&gt;HotKey&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
